src/flight_computer.py

```diff
@@ -74,20 +77,25 @@ def beep(duration = 0.2):
         GPIO.output(19, GPIO.LOW)
 
 
-    def vec_len(v):
-        return np.sqrt(np.dot(v, v))
+    # def vec_len(self, v):
+    #     return np.sqrt(np.dot(v, v))
```

Comment:
Why is this commented

Reply:
Because acceleration is a 1d vector at this point im pretty sure